python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -236,32 +243,69 @@ def read_fn():
         # Should be set to true in situations where another component, such as
         # a Kubernetes operator, is responsible for Ray setup on nodes.
         self.disable_node_updaters = self.config["provider"].get(
-            "disable_node_updaters", False
+            DISABLE_NODE_UPDATERS_KEY, False
         )
+        logger.info(f"{DISABLE_NODE_UPDATERS_KEY}:{self.disable_node_updaters}")
+        if self.disable_node_updaters:
+            logger.info("Node updaters are disabled.")
+        else:
+            logger.info("Node updaters are enabled.")
 
         # Disable launch config checking if true.
         # This is set in the fake_multinode situations where there isn't any
         # meaningful node "type" to enforce.
         self.disable_launch_config_check = self.config["provider"].get(
-            "disable_launch_config_check", False
+            DISABLE_LAUNCH_CONFIG_CHECK_KEY, False
+        )
+        logger.info(
+            f"{DISABLE_LAUNCH_CONFIG_CHECK_KEY}:{self.disable_launch_config_check}"
         )
+        if self.disable_launch_config_check:
+            logger.info("Launch config checks are disabled.")
```

Comment:
This is already covered by the logging
```
logger.info(
            f"{DISABLE_LAUNCH_CONFIG_CHECK_KEY}:{self.disable_launch_config_check}"
        )
```
right? Same for the other logging statement above :)

Reply:
Oh, good point. The English sentence doesn't really add much color for anyone.